deploy/aws/cloudbeat-aws.yml

```diff
@@ -43,6 +43,9 @@ output.elasticsearch:
 
   # Enable to allow sending output to older ES versions
   allow_older_versions: true
+  ssl:
+    certificate_authorities:
+      - ${ES_CERT}
```

Comment:
when using with elastic-package, the value will be at:
`~/.elastic-package/profiles/default/certs/ca-cert.pem`

Reply:
Have you also checked the cloud deployment? any adjuments required?

Reply:
when running cloudbeat locally with this and setting `ES_HOST` to something like `https://test-certs.kb.us-west2.gcp.elastic-cloud.com:443`, does it work for you without defining anything related to `ssl` ? asking because when i run it locally against elastic package i need to set `ssl.certificate_authorties` and it doesn't work without it, which turns out is the same (for me) when running against a remote `es`